partner_tag_smart_assignation/cron/update_cron.xml

```diff
@@ -1,14 +1,26 @@
 <odoo>
     <record id="smart_tags_updater" model="ir.cron">
         <field name="name">Smart Tags Updater</field>
-        <field name="active" eval="True" />
-        <field name="user_id" ref="base.user_root" />
+        <field name="active" eval="True"/>
+        <field name="user_id" ref="base.user_root"/>
         <field name="interval_number">1</field>
         <field name="interval_type">weeks</field>
         <field name="numbercall">-1</field>
         <field name="doall">1</field>
-        <field name="model_id" ref="model_res_partner_category" />
+        <field name="model_id" ref="model_res_partner_category"/>
         <field name="state">code</field>
         <field name="code">model.update_all_smart_tags()</field>
     </record>
+
+    <record id="ir_cron_check_validity_dates" model="ir.cron">
+        <field name="name">Check Validity Dates</field>
+        <field name="active" eval="True"/>
+        <field name="user_id" ref="base.user_root"/>
+        <field name="interval_number">1</field>
+        <field name="interval_type">days</field>
+        <field name="numbercall">-1</field>
+        <field name="model_id" ref="model_res_partner_category"/>
+        <field name="state">code</field>
+        <field name="code">model._check_validity_dates()</field>
+    </record>
```

Comment:
Instead of creating a new CRON, I think we could simply add your code inside the existing CRON which would perform the two tasks : update the tags to the people and/or deactivate the expired tags. What do you think?

Reply:
It would make more sense